README.md

```diff
@@ -618,6 +647,10 @@ func SyncExternalState() *reconcilers.SubReconciler[*resources.MyResource] {
 }
 ```
 
+#### Null
+
+[`Null`](https://pkg.go.dev/reconciler.io/runtime/reconcilers#Null) implements the SubReconciler interface, while performing no action. It can be used where a sub reconciler is required, but no behavior is necessary.
```

Comment:
**[question]:** Is there a conceivable _example_ like we've got for the others?

Reply:
Dropping null support for now. It's not essential to this PR.